public/_posts/_newsletters/Yearn-Finance-Newsletter-60/es.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: "Newsletter #60 de Yearn Finance"
+categories: [Newsletters]
+image:
+  src: ./cover.jpg
+  width: 576
+  height: 288
+author: Yearn
+date: '2022-04-13'
+translator: RestlessMik3
+---
+
+### Semana que finaliza el 12 de abril de 2022
+
+![](./image1.jpg?w=900&h=453)
+
+Bienvenido a la 60.ª edición de la Newsletter de Yearn Finance. Nuestro objetivo con esta Newsletter es mantener a Yearn y a la comunidad cripto informada sobre las últimas noticias, incluidos los lanzamientos de productos, los cambios en la gobernanza y las actualizaciones del ecosistema. Si está interesado en obtener más información sobre Yearn Finance, siga nuestras cuentas oficiales [Twitter](https://twitter.com/iearnfinance) y [Medium](https://medium.com/iearn).
+
+## Resumen
+
+- Anhelando Yearn
+- La estandarización de la Gran Vault
+- Cómo Web3 permite un Yearn sin censura
+- Cómo nacen las yVault Stratocaster 
+- Consejos para nuevos colaboradores del DAO
+- Vaults en Yearn
+- Noticias del ecosistema
+
+# Anhelando Yearn
+
+![](./image2.jpg?w=1000&h=563)
+
+Un informe premium sobre Yearn de Messari ya está disponible para todos. Originalmente compartida con sus suscriptores de pago en noviembre, la pieza contiene una gran cantidad de datos y explicaciones.
+
+Extractos del artículo dicen: "Todos conocemos a Yearn. Es el agregador de rendimiento que se configura y se olvida y se ejecuta sobre los protocolos de generación de rendimiento de DeFi como Compound, Aave, Curve y Convex. Los usuarios pueden ganar de forma pasiva con el clic de un botón Con el tiempo, el protocolo se ha convertido en sinónimo de rendimiento y se ha ganado una reputación en la industria como una fuente líder de rendimiento con riesgo minimizado... El protocolo insignia utilizado por la mayoría de los usuarios, Vaults representa el 67% del TVL de Yearn. una Vault que se define por activo y las estrategias de rendimiento subyacentes se ejecutan para implementar el capital... Yearn puede ofrecer un punto de integración único y confiable a otros protocolos de rendimiento para que no se pierda tiempo y no se introduzca el riesgo de integración administrando y manteniendo estrategias en sí mismas... A partir de aquí, el crecimiento de Yearn depende en gran medida de su interfaz de usuario V3, su estrategia multicadena y la defensa de su posición”.
```

Comment:
AS-IS: "Extractos del artículo dicen: "Todos conocemos a Yearn. Es el agregador de rendimiento que se configura y se olvida y se ejecuta sobre los protocolos de generación de rendimiento de DeFi como Compound, Aave, Curve y Convex. Los usuarios pueden ganar de forma pasiva con el clic de un botón Con el tiempo, el protocolo se ha convertido en sinónimo de rendimiento y se ha ganado una reputación en la industria como una fuente líder de rendimiento con riesgo minimizado... El protocolo insignia utilizado por la mayoría de los usuarios, Vaults representa el 67% del TVL de Yearn. una Vault que se define por activo y las estrategias de rendimiento subyacentes se ejecutan para implementar el capital... Yearn puede ofrecer un punto de integración único y confiable a otros protocolos de rendimiento para que no se pierda tiempo y no se introduzca el riesgo de integración administrando y manteniendo estrategias en sí mismas... A partir de aquí, el crecimiento de Yearn depende en gran medida de su interfaz de usuario V3, su estrategia multicadena y la defensa de su posición”. "


TO-BE: "Extractos del artículo dicen: "Todos conocemos a Yearn. Es el agregador de rendimiento que se configura, se olvida y se ejecuta sobre los protocolos de generación de rendimiento de DeFi como Compound, Aave, Curve y Convex. Los usuarios pueden ganar de forma pasiva con el clic de un botón. Con el tiempo, el protocolo se ha convertido en sinónimo de rendimiento y se ha ganado una reputación en la industria como una fuente líder de rendimiento con riesgo minimizado... El protocolo insignia utilizado por la mayoría de los usuarios, Vaults representa el 67% del TVL de Yearn. Una Vault que se define por activo y las estrategias de rendimiento subyacentes se ejecutan para implementar el capital... Yearn puede ofrecer un punto de integración único y confiable a otros protocolos de rendimiento para que no se pierda tiempo y no se introduzca el riesgo de integración administrando y manteniendo estrategias en sí mismas... A partir de aquí, el crecimiento de Yearn depende en gran medida de su interfaz de usuario V3, su estrategia multicadena y la defensa de su posición”.

Reply:
Done